apps/vmq_enhanced_auth/src/vmq_enhanced_auth.erl

```diff
@@ -209,6 +210,18 @@ init() ->
         ?TABLES
     ).
 
+set_acl_version_metrics(File) ->
+    case vmq_util:extract_version(File) of
+        Version when is_list(Version) ->
+            vmq_metrics:update_config_version_metric({acl_version, [{version, Version}]});
+        nomatch ->
+            vmq_metrics:update_config_version_metric({acl_version, [{version, "N/A"}]}),
+            error_logger:error_msg("no valid acl version found in the file ~p", [File]);
```

Comment:
Make this a warning message instead of error level or maybe remove this log.

Reply:
Removed log